bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/CreateBuilder.java

```diff
@@ -91,6 +92,15 @@
      */
     CreateBuilder withDigestType(DigestType digestType);
 
+    /**
```

Comment:
I think in the BP, we are going to support `CreateBuilder withWriteFlags(WriteFlag... writeFlags)`, no?

Reply:
ah my bad, You told me on slack ONLY to use EnumSet, I missed it on the wiki page.
I will add it

Reply:
no on slack, there are two methods, the varargs method can use the EnumSet method.

Reply:
done